packages/amplication-git-utils/src/providers/bitbucket/errors.ts

```diff
@@ -0,0 +1,13 @@
+export class BitbucketGenericError extends Error {
+  constructor(message: string, options: Record<string, unknown> = {}) {
+    super(message);
+    Object.assign(this, { ...options });
+    Object.setPrototypeOf(this, BitbucketGenericError.prototype);
+  }
+}
+export class BitbucketNotFoundError extends BitbucketGenericError {
+  constructor() {
+    super("404 Not found");
+    Object.setPrototypeOf(this, BitbucketNotFoundError.prototype);
+  }
+}
```

Comment:
Nice!
we also have this one: https://github.com/amplication/amplication/blob/master/packages/amplication-git-utils/src/utils/custom-error.ts
so, maybe we can delete it and use what you did, just add another class/es for the other errors